appinfo/info.xml

```diff
@@ -15,7 +15,7 @@ OpenID Connect has become the leading standard for single sign-on and identity p
 - Centralized client management within the Identity Provider
 - Enterprise-grade security through the use of authentication security features (e.g., multi-factor authentication) and policies (e.g., automatic token expiration on certain conditions) provided by Identity Providers
 
-The integration requires an Identity Provider which supports OpenID Connect (e.g., Keycloak, Kopano Konnect, Ping Federate, ADFS or others).
+The integration requires an Identity Provider that supports OpenID Connect (e.g., Keycloak, Kopano Konnect, Ping Federate, ADFS, Azure AD or others).
```

Comment:
wrong - azure ad will come with 1.1.0 - it is not part of 1.0.0

Reply:
@pmaier1 FYI

Reply:
True. azure support is not yet there. But it is an 'e.g.' list. Good enough, I'd say.

Reply:
Yes. This is intended. Thanks for the hint, though :)

Reply:
:shrug: